### DISCUSSION TITLE
Errors running bunch

### DISCUSSION THREAD

**Discussion by xtianmason** [ttscoff/bunch-docs#129] [Q&A] [answered]
When I start my bunch it runs, but I get two error messages that I Jane to click the ok button in order for the bunch to complete. One is on the title, the other on trying to set the order. I think I have the syntax right but I could be wrong. ![image](https://user-images.githubusercontent.com/51301235/128279768-6d5431d1-b351-4711-a4ae-53ae47ba2158.png)

![image](https://user-images.githubusercontent.com/51301235/128279796-c73a490f-dbc7-4f6a-9202-cc4941d7ce41.png)

**Comment by ttscoff** [comment, marked as answer]
You need three dashes before and after your frontmatter. Can you show me your bunch?

- Brett

**Reply by xtianmason** [reply]
Sure! Here is the text of it:

```
--
title: Work
menu order: 1
--
Music
Mail
Drafts
Things
Safari
CARROTweather
Slack
Shift
Messages
Grasshopper
* Say "Let's go to work"
Grasshopper_
Mail_
CARROTweather_
(audio output HP 24mh)
(audio input HD Webcam C615)

* tell application "Moom" to arrange windows according to snapshot "Work" ~10
```

Or if necessary I could send you the file.

Also, when saving the bunch it created another file in the folder with the same name at the beginning but then a string of letters and numbers. I deleted that file thinking it was an error, but maybe I was supposed to leave it alone?

**Reply by ttscoff** [reply]
It looks like you have two dashes, not three.

**Reply by ttscoff** [reply]
Regarding that extra file, I'm guessing your editor created that, not Bunch. Other than writing Example.bunch to your Bunch folder initially, Bunch doesn't create files.

**Reply by xtianmason** [reply]
Okay then it sounds like I just need to add a third dash to the front matter. I miscounted when looking at examples and instructions. Thanks!

**Reply by xtianmason** [reply]
So I changed the dashes to three and tried again. The bunch icon opens music right away, but then turns yellow for about a minute, then green and finishes. It did not give me the error messages.

**Comment by ttscoff** [comment]
Can you turn on debug logging and show your log when opening the Bunch?

- Brett
